engine/lib/src/platform_dispatcher.dart

```diff
@@ -574,9 +574,7 @@ class PlatformDispatcher {
         final int microseconds = _frameTime.inMicroseconds;
         _frameTime += const Duration(milliseconds: 16);
         Timer.run(() {
-          final Stopwatch beginSw = Stopwatch()..start();
-          _beginFrame(microseconds);
-          beginSw.stop();
+          late final Stopwatch beginSw;
```

Comment:
Don't think it needs to be `late`. Allocate the `Stopwatch` here, then just start it later. Or start it here too, the only difference is the time taken to to call `Timer.run` to schedule the time.
(So only move the `_beginFrame` call to after scheduling the timer, not the stopwatch allocation and start.)

Reply:
Addressed your comment in copybara-made gerrit CL & added you as reviewer

-> https://dart-review.googlesource.com/c/flute/+/399380